WebAPI/appsettings.json

```diff
@@ -20,15 +20,15 @@
     "Password": "<PASSWORD>"
   },
 
-  "IdentitySettings": {
-    "RequireDigit": "true",
-    "RequiredLength": "8",
-    "RequireNonAlphanumeric": "false",
-    "RequireUppercase": "true",
-    "RequireLowercase": "true",
-    "RequireConfirmedEmail": "true",
-    "DefaultLockoutTimeSpan": "30",
-    "MaxFailedAccessAttempts": "10",
-    "RequireUniqueEmail": "true" 
+    "IdentitySettings": {
```

Comment:
undo this change

Reply:
still not resolved

Reply:
![image](https://user-images.githubusercontent.com/26389312/48800148-3f763480-ed09-11e8-810d-6e860b222313.png)
it is just bad compare I remove only MailGuns section